Template/azuredeploy.json

```diff
@@ -36,6 +36,28 @@
             "metadata": {
                 "description": "In what region is your gateway deployed?"
             }
+        },
+        "spiSpeed": {
+            "type": "int",
+            "allowedValues": [
```

Comment:
ok, from here it really looks like 8 and 2 are the only allowed values... and the same for spiDev down there, 1 & 2